src/App.scss

```diff
@@ -29,6 +29,12 @@ a {
   text-decoration: none;
 }
 
+input, 
+textarea {
+  border: none;
+  outline: none;
```

Comment:
please be careful when resetting "outline" prop and make sure that user without mouse can determine when he is focused on your inputs
https://developer.mozilla.org/en-US/docs/Web/CSS/outline#accessibility_concerns